pym/bob/scm/git.py

```diff
@@ -72,6 +73,10 @@ def __init__(self, spec, overrides=[], secureSSL=None):
                     raise ParseError("Invalid remote name: " + stripped_key)
                 self.__remotes.update({stripped_key : val})
         self.__sslVerify = spec.get('sslVerify', secureSSL)
+        self.__recursive = ""
+        self.__recursive = spec.get('recursive', self.__recursive)
```

Comment:
Just do a `self.__recursive = spec.get('recursive', False)`. The conversion to the command line switch should be done in the `asScript` method.